docs/BR.md

```diff
@@ -1756,153 +1754,1157 @@ The CA SHALL enforce multi-factor authentication for all accounts capable of dir
+
+If the `subject` field of the certificate is an empty SEQUENCE, this extension MUST be marked critical, as specified in [RFC 5280, Section 4.2.1.6](https://tools.ietf.org/html/rfc5280#section-4.2.1.6). Otherwise, this extension MUST NOT be marked critical.
+
+Table: `GeneralName` within a `subjectAltName` extension
+
+| __Name Type__               | __Permitted__ | __Validation__ |
+| ---                         | -             | ------         |
+| `otherName`                 | N             | -              |
+| `rfc822Name`                | N             | -              |
+| `dNSName`                   | Y             | MUST contain the Fully-Qualified Domain Name. The Issuing CA MUST confirm that the Applicant controls or has been granted the right to use the Domain Name through a method specified in [Section 3.2.2.4](#3224-validation-of-domain-authorization-or-control). As an exception to the requirement of RFC 5280 requirements, Wildcard FQDNs are permitted. All other domain labels MUST be in the "preferred name syntax" and thus MUST NOT contain underscore characters ("\_"). MUST NOT contain an Internal Name. |
+| `x400Address`               | N             | -              |
+| `directoryName`             | N             | -              |
+| `ediPartyName`              | N             | -              |
+| `uniformResourceIdentifier` | N             | -              |
+| `iPAddress`                 | Y             | MUST contain the IPv4 or IPv6 address that the CA has confirmed the Applicant controls or has been granted the right to use through a method specified in [Section 3.2.2.5](#3225-authentication-for-an-ip-address). MUST NOT contain a Reserved IP Address. |
+| `registeredID`              | N             | -              |
+
+#### 7.1.2.8 OCSP Responder Certificate Profile
+
+If the Issuing CA does not directly sign OCSP responses, it MAY make use of an OCSP Authorized Responder, as defined by [RFC 6960](https://tools.ietf.org/html/rfc6960#section-4.2.2.2). The Issuing CA of the Responder MUST be the same as the Issuing CA for the Certificates it provides responses for.
+
+| __Field__                      | __Description__ |
+| ---                            | ------          |
+| `tbsCertificate`               | |
+| \ \ \ \ `version`              | MUST be v3(2) |
+| \ \ \ \ `serialNumber`         | MUST be a non-sequential number greater than zero (0) and less than 2^159 containing at least 64 bits of output from a CSPRNG. |
+| \ \ \ \ `signature`            | See [Section 7.1.3.2](#7132-signature-algorithmidentifier) |
+| \ \ \ \ `issuer`               | MUST be byte-for-byte identical to the `subject` field of the Issuing CA. See [Section 7.1.4.1](#7141-name-encoding) |
+| \ \ \ \ `validity`             | See [Section 7.1.2.8.x](#7128x-ocsp-responder-validity) |
+| \ \ \ \ `subject`              | See [Section 7.1.2.10.1](#712101-ca-certificate-naming) |
```

Comment:
Is it intentional for the OCSP responder certificate profile to require it to meet CA certificate naming requirements for Subject?

Reply:
Looking at current OCSP responder certificates on Censys, a number of them does not include a unique identifier so just highlighting this may be a change to current practices for some CAs. RFC5280 states "A CA MAY issue more than one certificate with the same DN to the same subject entity." Would OCSP responders be considered "the same subject entity"?

Reply:
I'm not sure I fully grok the 'unique identifier' comment.

At present, fully unique identifiers aren't required AIUI. That is, the only requirement close to that is in the `commonName`, namely
> The contents SHOULD be an identifier for the certificate such that the certificate's Name is unique across all certificates issued by the issuing certificate.

But that's as a SHOULD (aka RECOMMENDS), but not a MUST/SHALL normative requirement. Thus, pregenerating OCSP responders with same name & key would still be permitted (at least, within the scope of 7.1.2.8.1, which may change in a future/new ballot, but is presently unrestricted)

Reply:
Yes it did, thank you.